tests/CoreFX/CoreFX.issues.json

```diff
@@ -1,306 +1,415 @@
-﻿[
-    {
-        "name": "System.Data.SqlClient.ManualTesting.Tests",
-        "enabled": false,
-        "exclusions": {
-            "namespaces": null,
-            "classes": null,
-            "methods": null
+﻿
+
+    [
```

Comment:
Is the indentation change for the whole file intentional?

Reply:
Not at all. My linter seems to have gotten away from me. Changed.